index.js

```diff
@@ -300,7 +306,7 @@ function npmbin (name) {
 }
 
 function shell () {
-  return os.platform() === 'android' ? 'sh' : undefined
+  return os.platform() === 'android' ? 'sh' : true
```

Comment:
We only need it on Windows, on other platforms we can avoid the overhead of a shell.